lang/zh_CN.lang

```diff
@@ -561,13 +657,66 @@ value.WATER_FOG.1=始终可见
 option.WATER_FOG_DENSITY=水雾密度
 option.WATER_FOG_DENSITY.comment=调整水雾密度。
 
-option.WATER_CAUSTICS_STRENGTH=焦散强度
-option.WATER_CAUSTICS_STRENGTH.comment=调整水焦散的强度。
+option.WATER_CAUSTICS_STRENGTH=光晕强度
```

Comment:
looking at google translate, old is "caustics" while the new one is just "glow". please recheck.

Reply:
changed